.github/workflows/1_2_b_bump_extension_only.yml

```diff
@@ -0,0 +1,71 @@
+name: Bump versions for extension only (on push to master and patch branch)
+
+# This workflow covers both '1_check_for_updates' and '2_bump_versions' workflows, therefore it is another variant of the same, but specifically for extension only, without any Prisma CLI updates that would need to be handled.
+
+on:
+    push:
+      branches: 
+        - 'master'
+        - '**.x'
+      paths-ignore:
+        - '.github/**'
```

Comment:
Why is this required?

Reply:
It's not required. 

I thought it would be good to have. If we push sth that only changes the workflow files, we don't really need to publish a new version.

Reply:
But what if we are fixing something in the workflow itself?

Reply:
What do you mean? If we are fixing something in the workflow it will get pushed to master, but we would not automatically release a new Insider version just because of that.

Reply:
I meant what if we want to fix the workflow to add a feature in the insiders but a minor point. It is okay for this condition to be here 👍

Reply:
Ah got it 👍